File: KingdomCodeTool/wwwroot/Download/202410291354/Service/Implement/DanhMucCongViecService.cs
namespace Service.Implement
{
    public class DanhMucCongViecService : BaseService<DanhMucCongViec, IDanhMucCongViecRepository>
    , IDanhMucCongViecService
    {
    private readonly IDanhMucCongViecRepository _DanhMucCongViecRepository;
    public DanhMucCongViecService(IDanhMucCongViecRepository DanhMucCongViecRepository) : base(DanhMucCongViecRepository)
    {
    _DanhMucCongViecRepository = DanhMucCongViecRepository;
    }
    }
    }

File: KingdomCodeTool/wwwroot/Download/202410291354/Angular/shared/DanhMucCongViec.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucCongViec } from 'src/app/shared/DanhMucCongViec.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucCongViecService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucCongViec";
    }
}

